multinet/db.py

```diff
@@ -608,3 +608,44 @@ def count_text(filt: str) -> str:
         "edges": list(aql_query(workspace, query)),
         "count": next(aql_query(workspace, count)),
     }
+
+
+@lru_cache(maxsize=1)
+def uploads_collection() -> StandardCollection:
+    """Return the collection used for temporarily storing uploaded file chunks."""
+    sysdb = db("_system")
+
+    if not sysdb.has_collection("uploads"):
+        sysdb.create_collection("uploads")
+
+    return sysdb.collection("uploads")
+
+
+def create_upload_document() -> Any:
+    """Insert empty multipart upload temp document."""
+    # returns arango document key as upload_id for client
+    return uploads_collection().insert({}).get("_key")
+
+
+def insert_file_chunk(key: str, sequence: str, chunk: str) -> str:
+    """Insert b64-encoded string `chunk` into temporary document."""
+    collection = uploads_collection()
+
+    document = collection.get(key)
+    if document is None:
+        raise NonExistantUploadDocument()
+
+    document[sequence] = chunk
+    collection.update(document)
+
+    return key
+
+
+def delete_document(collection: StandardCollection, key: str) -> Dict:
+    """Delete a document in the given collection with the specified key."""
+    document = collection.get({"_key": key})
+    if document is None:
+        raise NonExistantUploadDocument()
```

Comment:
The error refers to upload documents, but the function is general purpose.

It seems like this should be called `delete_upload_document()` and be specialized to just those, right?